src/node.js

```diff
@@ -208,6 +208,25 @@
         hrValues[2]
       ];
     };
+
+    const _cpuUsage = process.cpuUsage;
+    const cpuValues = new Uint32Array(8);
+
+    process.cpuUsage = function cpuUsage(ar) {
```

Comment:
Woops, leftover garbage from copying `hrtime()` bits, above this.  Should **not** take any arguments (`ar` should be removed).